src/ts/components/seekbar.ts

```diff
@@ -253,6 +259,10 @@ export class SeekBar extends Component<SeekBarConfig> {
       if (this.isUiShown) {
         this.setAriaSliderValues();
       }
+
+      if (typeof uimanager.getConfig().seekbarSnappingRange === 'number') {
+        this.config.snappingRange = uimanager.getConfig().seekbarSnappingRange;
+      }
```

Comment:
I didn't see it the first time, but we should not need to do this on every playback position update.

I would suggest moving it to after the marker config initializiation in line 389:

```
    // Add markers when a source is loaded or update when a marker is added or removed
    uimanager.getConfig().events.onUpdated.subscribe(() => {
      playbackPositionHandler();
    });
```

Reply:
Good point, I didn't realise that. I've moved the code. Thanks!